framework-cloud/framework-openshift-strimzi/src/main/java/org/kie/cloud/strimzi/TopicOperator.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.cloud.strimzi;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.kie.cloud.api.scenario.HACepScenario;
+import org.kie.cloud.openshift.resource.Project;
+import org.kie.cloud.openshift.resource.impl.ProjectImpl;
+
+public class TopicOperator {
+    private final String DEFAULT_TOPIC_NAME = "my-topic";
+    private final String KAFKA_TOPIC_EXAMPLE = "examples/topic/kafka-topic.yaml";
+
+    private HACepScenario HACepScenario;
+
+    public TopicOperator(final HACepScenario HACepScenario) {
+        this.HACepScenario = HACepScenario;
+    }
+
+    public void createTopic(final String name) {
+        final File topicYamlFile = new File(HACepScenario.getAMQStreamsDirectory(), KAFKA_TOPIC_EXAMPLE);
+        String topicYaml;
+        try {
+            topicYaml = FileUtils.readFileToString(topicYamlFile);
```

Comment:
Could be replaced by Files.readString(path)

Reply:
Isn't readString in Java 11 and higher? This is still Java 1.8 code.

Reply:
Good point, I overlooked it. I personally trying to avoid using external libraries when usable JDK alternative is available (to avoid dependency hell). Anyway the readFileToString() is deprecated, please use suggested replacement.